docs/config.md

```diff
@@ -680,6 +688,56 @@ profile_models:
   arguments in this section. An example of this is `http-port`, which is an
   argument to Model Analyzer itself.
 
+### `<triton-server-environment>`
+
+This section enables setting environment variables for the tritonserver
+instances launched by model-analyzer. This is useful, for examples, when
+a custom operation is required by a model and Triton requires the LD_PRELOAD and
+LD_LIBRARY_PATH environment variables to be set. The section is a dictionary where the
```

Comment:
"The section is a dictionary where the" -> "The value for this section is a dictionary"..